api/wan/read

```diff
@@ -78,7 +78,11 @@ if($cmd eq 'providers') {
         $ret->{'configuration'}{'multiwan'}{$_} = $cdb->get_prop('firewall',$_);
     }
     $ret->{'configuration'}{'multiwan'}{'EmailAddress'} = $cdb->get_prop('root', 'EmailAddress');
-
+    if ($ret->{'configuration'}{'multiwan'}{'WanMode'} eq 'backup'){
+		my $externalip = `dig +short myip.opendns.com \@resolver1.opendns.com`;
+		$ret->{'configuration'}{'multiwan'}{'CurrentIp'} = substr($externalip, 0, length($externalip)-1);
+	}
```

Comment:
Cosmetic comment on code indentation, align the ` {` parenthesis to the parent `if`

Reply:
Changed.